src/AccessibilityInsights.SharedUx/Controls/ScannerResultControl.xaml

```diff
@@ -34,118 +35,11 @@
-                                                    </TextBlock>
-                                                </StackPanel>
-                                            </DataTemplate>
-                                        </GridViewColumn.CellTemplate>
-                                    </GridViewColumn>
-                                    <GridViewColumn Width="{Binding BugColumnWidth}">
-                                        <GridViewColumnHeader 
-                                            Style="{DynamicResource ScannerResultColumnHeader}"
-                                            Visibility="{Binding BugColumnVisibility}" 
-                                            Content="{x:Static Properties:Resources.ScannerResultControl_Issue}" 
-                                            IsTabStop="False" 
-                                            Focusable="True"/>
-                                        <GridViewColumn.CellTemplate>
-                                            <DataTemplate>
-                                                <StackPanel Orientation="Horizontal" MinWidth="200">
-                                                    <fabric:FabricIconControl GlyphName="LadybugSolid" GlyphSize="Custom" FontSize="15" HorizontalAlignment="Center" VerticalAlignment="Center" Foreground="{DynamicResource ResourceKey=BlueButtonBGBrush}"
-                                                                              Margin="0,0,4,0"  Visibility="{Binding Path=FileBugVisibility}"/>
-                                                    <Button Style="{StaticResource ResourceKey=btnLink}" Name="btnFileBug" Click="btnFileBug_Click"
-                                                    Content="{Binding IssueDisplayText, TargetNullValue='{x:Static Properties:Resources.btnFileIssueContent}'}"
-                                                    Margin="-1" Focusable="True" IsTabStop="True" Tag="{Binding }" HorizontalAlignment="Center"
-                                                    AutomationProperties.HelpText="{Binding Path=Element.Glimpse, StringFormat='{x:Static Properties:Resources.btnFileBugAutomationPropertiesHelpText1}'}"
-                                                    AutomationProperties.Name="{Binding Path=BugIdString, StringFormat='Open link to bug #{0}', TargetNullValue='{x:Static Properties:Resources.btnFileIssueContent}'}"
-                                                    Visibility="{Binding Path=FileBugVisibility}"/>
-                                                </StackPanel>
-                                            </DataTemplate>
-                                        </GridViewColumn.CellTemplate>
-                                    </GridViewColumn>
-                                </GridView>
-                            </ListView.View>
-                        </ListView>
+                            Grid.IsSharedSizeScope="True" KeyboardNavigation.TabNavigation="Once"/>
```

Comment:
Do these belong on the custom control or on the ListView? The TabNavigation in particular feels like it might belong on the ListView